ceph_deploy/hosts/__init__.py

```diff
@@ -6,17 +6,17 @@
 """
 import logging
 from ceph_deploy import exc
-from ceph_deploy.hosts import debian, centos, fedora, suse, remotes
+from ceph_deploy.hosts import debian, centos, fedora, suse, remotes, rhel
 from ceph_deploy.connection import get_connection
 
 logger = logging.getLogger()
 
 
-def get(hostname, username=None, fallback=None, detect_sudo=True):
+def get(hostname, args, fallback=None, detect_sudo=True):
```

Comment:
@trhoden Is there a reason we are accepting args here instead of expanding the explicit kwargs to include 'use_rhceph' ?